Makefile

```diff
@@ -216,8 +245,53 @@ zipfile: dist
 	(cd zipfile && zip -r $(PKG).zip $(PKG)/)
 	@echo "ZIP file is ready at zipfile/$(PKG).zip"
 
+version-update:
+	@echo -e "\nUpdating $(RPM_SPEC) version, release, and changelog..."
+	sed -e s/__NAME__/$(PKGNAME)/ \
+		$(RPM_SPEC).in > $(RPM_SPEC)
+	sed -i s/__VERSION__/$(SSG_VERSION)/ \
+		$(RPM_SPEC)
+	sed -i s/__RELEASE__/$(SSG_RELEASE_VERSION)/ \
+		$(RPM_SPEC)
+	sed -i 's/__DATE__/$(SSG_RELEASE_DATE)/' \
+		$(RPM_SPEC)
+	sed -i 's/__REL_MANAGER__/$(SSG_REL_MANAGER)/' \
+		$(RPM_SPEC)
+	sed -i 's/__REL_MANAGER_MAIL__/$(SSG_REL_MANAGER_MAIL)/' \
+		$(RPM_SPEC)
+
+srpm: tarball version-update rpmroot
+	cat $(RPM_SPEC) > $(RPMBUILD)/SPECS/$(notdir $(RPM_SPEC))
+	cp tarball/$(PKG).tar.gz $(RPMBUILD)/SOURCES/
+	@echo -e "\nBuilding $(PKGNAME) SRPM..."
+	cd $(RPMBUILD) && rpmbuild $(RPMBUILD_ARGS) --target=$(ARCH) -bs SPECS/$(notdir $(RPM_SPEC)) --nodeps
+
+rpm: srpm
+	@echo -e "\nBuilding $(PKGNAME) RPM..."
+	cd $(RPMBUILD)/SRPMS && rpmbuild --rebuild --target=$(ARCH) $(RPMBUILD_ARGS) --buildroot $(RPMBUILD)/BUILDROOT -bb $(PKG)-$(SSG_RELEASE_VERSION)$(OS_DIST).src.rpm
+
+git-tag:
+	@echo -e "\nUpdating $(RPM_SPEC) changelog to reflect new release"
+	sed -i '/\%changelog/{n;s/__DATE__/$(RPM_DATESTR)/}' $(RPM_SPEC).in
+	sed -i '/\%changelog/{n;s/__REL_MANAGER__/$(SSG_REL_MANAGER)/}' $(RPM_SPEC).in
+	sed -i '/\%changelog/{n;s/__REL_MANAGER_MAIL__/$(SSG_REL_MANAGER_MAIL)/}' $(RPM_SPEC).in
+	sed -i '/\%changelog/{n;s/__VERSION__/$(SSG_VERSION)/}' $(RPM_SPEC).in
+	sed -i '/\%changelog/{n;s/__RELEASE__/$(SSG_RELEASE_VERSION)/}' $(RPM_SPEC).in
+	sed -i '/new/{s/__VERSION__/$(SSG_VERSION)/}' $(RPM_SPEC).in
+	sed -i '/\%changelog/a\* __DATE__ __REL_MANAGER__ <__REL_MANAGER_MAIL__> __VERSION__-__RELEASE__\n- Make new __VERSION__ release\n' $(RPM_SPEC).in
+	@echo -e "\nTagging $(PKGNAME) to new release $(NEW_RELEASE)"
+	$(eval NEW_RELEASE:=$(shell git describe $(git rev-list --tags --max-count=1) | awk -F . '{printf "%s.%i.%i", $$1, $$2, $$3 + 1}' | sed 's/^.//'))
+	$(eval NEW_MINOR_RELEASE:=$(shell echo $(NEW_RELEASE) | awk -F . '{printf "%i", $$3}'))
+	@echo -e "\nUpdating VERSION to new minor release $(NEW_RELEASE)"
+	sed -i 's/SSG_MINOR_VERSION.*/SSG_MINOR_VERSION = $(NEW_MINOR_RELEASE)/' $(ROOT_DIR)/VERSION
+	sed -i 's/SSG_RELEASE_DATE.*/SSG_RELEASE_DATE = $(RPM_DATESTR)/' $(ROOT_DIR)/VERSION
+	@echo -e "\nTagging to new release $(NEW_RELEASE)"
+	git add $(RPM_SPEC).in $(ROOT_DIR)/VERSION
+	git commit -m "Make new $(NEW_RELEASE) release"
+	git tag -a -m "Version $(NEW_RELEASE)" v$(NEW_RELEASE)
+
 clean:
-	rm -rf tarball/
+	rm -rf $(RPMBUILD)
```

Comment:
@mpreisler in reason as to why `rm -rf tarball` was removed? In fact, shouldn't there be a `rm -rf zipfile` as well?